api/time_bucket_ng.md

```diff
@@ -1,9 +1,9 @@
 ## timescaledb_experimental.time_bucket_ng() <tag type="experimental">Experimental</tag>
-
-The `time_bucket_ng()` (next generation) experimental function is
-similar to [`time_bucket()`][time_bucket], but works with years and
-months. It is expected that the feature will also support timezones
-in a future release.
+The `time_bucket_ng()` (next generation) experimental function is an updated
+version of  the original [`time_bucket()`][time_bucket] function. While
+`time_bucket` works with small units of time, the `time_bucket_ng()` function
```

Comment:
I do not think this is accurate. @afiskon Could you confirm if time_bucket_ng supports all the "intervals" supported by time_bucket + ( years  and months ) ?
@Loquacity See the comparison table between time_bucket and time_bucket_ng here: https://blog.timescale.com/blog/move-fast-but-dont-break-things-introducing-the-experimental-schema-with-new-experimental-features-in-timescaledb-2-4/

Reply:
time_bucket_ng() does support everything time_bucket() does. There could be a mistake in the blogpost though. Not sure what was meant by the support of timezones in time_bucket(). Last time I reviewed the draft the table looked different.

Reply:
cc: @ryanbooz